clients/csrf/default_csrf_token_fetcher.go

```diff
@@ -1,18 +1,20 @@
 package csrf
 
 import (
-	"github.com/cloudfoundry-incubator/multiapps-cli-plugin/clients/csrf/csrf_paramters"
-	"github.com/cloudfoundry/cli/plugin"
 	"net/http"
 	"os"
+
+	"github.com/cloudfoundry-incubator/multiapps-cli-plugin/clients/csrf/csrf_parameters"
+	"github.com/cloudfoundry-incubator/multiapps-cli-plugin/log"
+	"github.com/cloudfoundry/cli/plugin"
 )
 
 const CsrfTokenHeaderFetchValue = "Fetch"
 const CsrfTokensApi = "/api/v1/csrf-token"
 const ContentTypeHeader = "Content-Type"
 const AuthorizationHeader = "Authorization"
 const ApplicationJsonContentType = "application/json"
-const CookieHeader = "CookieHeader"
+const CookieHeader = "Cookie"
```

Comment:
Why this constant was necessary to be changed?

Reply:
This here was the actual issue - the header is called 'Cookie'. we were not properly clearing the header value before, because the header string was made-up.